src/rpc/handlers/AccountLines.hpp

```diff
@@ -136,6 +137,8 @@ class AccountLinesHandler {
              modifiers::Clamp<int32_t>{LIMIT_MIN, LIMIT_MAX}},
             {JS(ledger_index), validation::LedgerIndexValidator},
             {JS(marker), validation::AccountMarkerValidator},
+            {JS(ledger), check::Deprecated{}},
+            {"peer_index", check::Deprecated{}},
```

Comment:
I assume there is no JS string for this?

Reply:
No